### ISSUE TITLE
Need to copy twice on Terminator in order for the content to show in the extension

### ISSUE THREAD

**Issue by grizonic** [Tudmotu/gnome-shell-extension-clipboard-indicator#194]
I'm on Fedora 31 Wayland and been u extension for a while. I always felt something was off, but today I decided to try to find out what. There are missing items in the history and they are always the ones coming from terminator.

Basically everything works fine with all tools - terminal included - but while in terminator I need to ctrl+shift+c twice for the content to show up in the clipboard manager.

I saw in the issues list that you were trying (maybe already succeeding!) to support the auto copy on select functionality of terminator. Could that be conflicting with the basic copy?

Congrats on the great work here. Really helps my day to day :) Thank you!

**Comment by rmader** [comment]
As it turns out this is actually a GTK issue. We're about to fix it :)

**Comment by rmader** [comment]
This got fixed in https://gitlab.gnome.org/GNOME/gtk/merge_requests/1438, released in 3.24.14

**Comment by GregorKopka** [comment]
@rmader I can reproduce the issue in a X11 session.

What I found is that Clipboard Indicator seems to trigger (or cause?) what's described in https://gitlab.gnome.org/GNOME/mutter/issues/919#note_739406 in a quite boolean fashion: 

With extension enabled I can reliably reproduce selected data not being copied to the clipboard on basically every 2nd operation when using cut/paste to reorder rows in an embedded table in a writer document. With extension disabled I seem to be completely unable to reproduce the issue.

@terencode as your extension is basically able to toggle the presence of the issue (at least on my system), could you please take a look if you are able to locate how that is possible?

Tested on current Funtoo with
gnome-base/gnome-3.34.0:2.0
x11-wm/mutter-3.34.3
x11-libs/gtk+-3.24.13
app-office/libreoffice-6.2.4.2-r2

**Comment by Filbuntu** [comment]
Same behaviour mentioned by @GregorKopka happens on Ubuntu 19.10 with Clipboard Indicator:  
gnome-shell 3.34.2-2ubuntu1 ~19.10.1  
mutter 3.34.1+git20191107-1ubuntu1 ~19.10.1  
x11-common 1:7.7+19ubuntu12  
Libreoffice 6.3.5.2

**Comment by rmader** [comment]
If you are experiencing this on X11, you are (most likely) not running into the same bug as the the Author of the the issue - that bug has been fixed in **GTK 3.24.14**, which both of you apparently also do not have yet.

On X11, you might run into https://gitlab.gnome.org/GNOME/mutter/issues/1113 - we don't yet know what why that is.

**Comment by GregorKopka** [comment]
@rmader Yes, I'm one release behind on gtk+. It also might be that it's that other issue in mutter, but in that case... why does the issue only surface while having Clipboard Indicator active?

**Comment by kapouer** [comment]
Hi, on debian/sid, gnome on wayland, libgtk 3.24.14, i somewhat reproduce the issue: once copied, i need to select the item twice in clipboard history or else it doesn't paste anything.

**Comment by rmader** [comment]
> Hi, on debian/sid, gnome on wayland, libgtk 3.24.14, i somewhat reproduce the issue: once copied, i need to select the item twice in clipboard history or else it doesn't paste anything.

Yes, I can reproduce this, too. It's not the same bug though - mind opening a new issue?

**Comment by kapouer** [comment]
Yep, that'd be #198

**Comment by rmader** [comment]
@grizonic: mind confirming that this particular issue has been fixed? :)
